truffle/test/LoadContract.js

```diff
@@ -164,46 +163,60 @@ contract('LoadContract', async (accounts) => {
         assert.equal(data.shipment.carrier, CARRIER);
         assert.equal(data.shipment.moderator, MODERATOR);
         assert.equal(data.shipment.state, ShipmentState.CREATED);
-        assert.equal(data.shipment.vaultUri, "");
-        assert.equal(data.shipment.vaultHash, "");
     });
 
     it("should have a getShipmentData function, and work with createNewShipment2", async () => {
         const shipmentUuid = uuidToHex(uuidv4(), true);
-        await contract.createNewShipment2(shipmentUuid, EscrowFundingType.NO_FUNDING, 0, "uri", "hash", CARRIER, {from: SHIPPER});
+        await contract.createNewShipment2(shipmentUuid, EscrowFundingType.NO_FUNDING, 0,  CARRIER, {from: SHIPPER});
+        await notary.registerVault(shipmentUuid, "uri", "hash", {from: SHIPPER});
```

Comment:
There are no assertions around this line - did you forget to add one or is this line intentional?

Reply:
You mean to add assertions for uri and hash here? I was thinking those are tested in other unit tests in VaultNotary.js. We can discuss this Monday.

Reply:
No, I mean that this test calls `registerVault`, and I'm not sure why it's necessary for the test scenario here? I think this line can be removed unless you are making assertions about the notary here.